src/main/java/org/example/homework4/LeapYear.java

```diff
@@ -0,0 +1,15 @@
+package org.example.homework4;
+
+public class LeapYear {
+    public static void main(String[] args) {
+        int year = 2024;
+        if (isLeapYear(year)) {
+            System.out.printf("The year " + year + " is leap year");
```

Comment:
Why printf, if you don't use formatting? Use println or add any formatting logic
```suggestion
            System.out.printf("The year %04d is leap year%n", year);
```